### DISCUSSION TITLE
can not delete acount or scence

### DISCUSSION THREAD

**Discussion by m061i6** [mozilla/hubs#5402] [Hubs Cloud Questions] [answered]
i use the hubs cloud aws 
when i try to delete acount or scrnce
the page will be redirect to login page
i am the admin acount

**Comment by matthewbcool** [comment, marked as answer]
These are known issues we can follow here:
https://github.com/mozilla/hubs-cloud/issues/79
https://github.com/mozilla/hubs-cloud/issues/59

**Reply by techcto** [reply]
I am not sure these answers make anyone feel better. At first they give you hope, but then are left open ended.  Can we just mark this a bug since there is a delete button and it does not work?  Is anyone working on this? Do we just move on and leave the unwanted scenes?
